SU-SOE/stanford_soe_helper
# 7.x-2.x soe 3031

# READY FOR REVIEW

# Summary
People Spotlight: add styling to scss.

# Needed By (Date)
- Sprint end

# Criticality
- Getting this into next release


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-3031 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-3031)
- edit the injector: `admin/config/development/css-injector/edit/52` and disable it.
- edit the injector: `admin/config/development/css-injector/edit/54` and disable it.
- edit the injector: `admin/config/development/css-injector/edit/56` and disable it.
- edit the injector: `admin/config/development/css-injector/edit/58` and disable it.
- `drush cc css-js`
- go to: `/magazine/article/jeremy-bailenson-taking-grand-tour-latest-virtual-reality` and make sure the changes look like the screen shot below.

# Injector code and screen shots
`.node-type-stanford-magazine-article .group-s-mag-article-date-byline .field-name-field-s-mag-article-collection .field-label:after {
  content: "\00a0 \00a0";
}`

![screen shot 2018-06-19 at 5 50 35 pm](https://user-images.githubusercontent.com/571421/41628333-a51df97e-73e9-11e8-8e6f-3cc2bd43648a.png)

`.node-type-stanford-magazine-article .group-s-mag-article-date-byline .field-name-field-s-mag-article-collection .field-items {
  margin-top: 1px;
}

![screen shot 2018-06-19 at 6 05 18 pm](https://user-images.githubusercontent.com/571421/41628708-683e30da-73eb-11e8-9227-0964c40ca39c.png)

@media (max-width: 979px) {
  .node-type-stanford-magazine-article .group-s-mag-article-date-byline .field-name-field-s-mag-article-collection .field-label {
    font-size: 17px;
  }
  .node-type-stanford-magazine-article .group-s-mag-article-date-byline .group-s-social-and-print {
    margin-left: -13px;
  }
}

![screen shot 2018-06-19 at 6 08 10 pm](https://user-images.githubusercontent.com/571421/41628801-dafc5250-73eb-11e8-8c6e-fe6b79ace16c.png)

![screen shot 2018-06-19 at 6 02 02 pm](https://user-images.githubusercontent.com/571421/41628667-30d8bed0-73eb-11e8-8afb-edbd30d34cff.png)

@media (max-width: 767px) {
  .node-type-stanford-magazine-article .group-s-mag-article-date-byline .field-name-field-s-mag-article-collection .field-label {
    font-size: 16.5px;
  }
}

![screen shot 2018-06-19 at 6 02 18 pm](https://user-images.githubusercontent.com/571421/41628670-3826cd94-73eb-11e8-9da4-d85aab5e1985.png)

@media (max-width: 580px) {
  .node-type-stanford-magazine-article .group-s-mag-article-date-byline .field-name-field-s-mag-article-collection .field-label {
    font-size: 16.1px;
  }
}

![screen shot 2018-06-19 at 6 02 46 pm](https://user-images.githubusercontent.com/571421/41628673-3bbf25aa-73eb-11e8-920f-647e055b970f.png)

@media (max-width: 400px) {
  .node-type-stanford-magazine-article .group-s-mag-article-date-byline .field-name-field-s-mag-article-collection .field-label {
    font-size: 15px;
  }
}`

NO SCREEN SHOT: It's there though :D

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3031

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
